app/views/paginable/plans/_privately_visible.html.erb

```diff
@@ -42,6 +43,13 @@
                 <td class="plan-visibility">
                   <%= plan.visibility === 'is_test' ? _('N/A') : raw(display_visibility(plan.visibility)) %>
                 </td>
+                <td>
+                  <% unless plan.shared.empty? %>
+                    <%= _("Yes(#{plan.shared.size})") %>
```

Comment:
Use `.length` instead. `.size` or `.count` hit the DB whereas `.length` just counts the objects in the association array